### ISSUE TITLE
NullReferenceException when starting game

### ISSUE THREAD

**Issue by kabili207** [FelixReuthlinger/AutoMapPins#37]
This doesn't seem to affect things during gameplay, but I figured I should report it just in case

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
AutoMapPins.Model.PinComponent+<VisibleCheck>d__6.MoveNext () (at <09ad55f388a14e84bec1801566808d60>:0)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <3aefadeb43334fcf8b7b268355c16be2>:0)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
AutoMapPins.Model.PinComponent:SetVisiblePin()
AutoMapPins.Model.PinComponent:Awake()
UnityEngine.GameObject:AddComponent()
AutoMapPins.Patches.CommonPatchLogic:Patch(GameObject)
AutoMapPins.Patches.ContainerPatch:Postfix(Container&)
Container:DMD<Container::Awake>(Container)
UnityEngine.GameObject:AddComponent()
AdventureBackpacks.Patches.PlayerAwakePatch:Postfix(Player)
Player:DMD<Player::Awake>(Player)
UnityEngine.Object:Instantiate(GameObject, Vector3, Quaternion)
FejdStartup:SetupCharacterPreview(PlayerProfile)
FejdStartup:UpdateCharacterList()
FejdStartup:SetSelectedProfile(String)
FejdStartup:DMD<FejdStartup::Start>(FejdStartup)

```

**Comment by FelixReuthlinger** [comment]
how did you get that error?

**Comment by kabili207** [comment]
It happens during the initial game load, before the main screen shows up. I've attached the full log file if it helps
[LogOutput.log](https://github.com/user-attachments/files/16240959/LogOutput.log)

**Comment by FelixReuthlinger** [comment]
ok, not sure if this would fix it, but try version 2.2.1

**Comment by kabili207** [comment]
That seems to have resolved it
